GitTfs/Core/GitTfsRemote.cs

```diff
@@ -695,10 +703,17 @@ private void quickFetch(ITfsChangeset changeset)
         private IEnumerable<ITfsChangeset> FetchChangesets(bool byLots, int lastVersion = -1)
         {
             int lowerBoundChangesetId;
+
+            // If we're starting at the Root side of a branch commit (e.g. C1) but there are invalid commits between C1 and the actual branch-side of the commit operation
```

Comment:
It would be helpful if each comment line fit within the line-length for a GitHub pull request code file review pane.